src/utils/storeUniqueToken.ts

```diff
@@ -0,0 +1,39 @@
+import { env } from "../typeDefinitions/default.types";
+import jwt from "@tsndr/cloudflare-worker-jwt";
+
+export const storeUniqueToken = async (
+  token: string,
+  discordId: number,
+  env: env
+) => {
+  const jwtToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 15 },
+    env.PRIVATE_TOKEN,
```

Comment:
change the name to DISCORD_PRIVATE_KEY as that is how we will save it in GitHub env.

Please create a ticket to add the key to workflow so that it gets saved in staging bot's cloudflare worker.

Reply:
Sure, will do.
Thanks!